osuosl/climesync
# Added time summary to get_projects

fixes issue #65 
## Changes in this PR.


- [X] The `gp` command now fetches all times submitted under each project and prints out a short summary for each project
## Testing this PR.


1. Run `nosetests`
2. Run climesync, connect to the staging server, and login as "admin" with the password "timesync-staging" 
3. Run `gp`, and when asked for a project slug enter "px". You should see the fields "first_time", "latest_time", "num_times", and "time_total" inside the project.

@osuosl/devs
